src/Hyperwallet/Model/BankAccount.php

```diff
@@ -63,6 +63,7 @@
  * @property string $stateProvince The state or province
  * @property string $country The country
  * @property string $postalCode The postal code
+ * @property string $isDefaultTransferMethod The is default transfer method
```

Comment:
Suggestion: The field isDefaultTransferMethod can be explained better for the developers to understand. (Same for all model classes)

Reply:
There were some existing models which were using this property "isDefaultTransferMethod" and just to make it consistent across models, reused the same description. I've now updated the description for all of the models which were using this property